beard/clustering/blocking.py

```diff
@@ -37,6 +43,42 @@ def predict(self, X):
         return _single(X)
 
 
+def _parallel_fit(fit_, partial_fit_, base_estimator,
+                  clusterers_, block_name, X, y):
+
+    clusterers = {}
```

Comment:
now that we create one task per block, there is no need for this container. We can directly return `(block_name, clusterer)`